src/main/java/com/example/demo/entity/Employee.java

```diff
@@ -0,0 +1,68 @@
+package com.example.demo.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="emp")
+public class Employee 
+{
+	@GeneratedValue(strategy= GenerationType.AUTO)
+	@Id
+	private int id;
+	private String firstName;
+	private String lastName;
+	private String emailId;
+	
+	public Employee() {
+	}
+
+	public Employee(int id, String firstName, String lastName, String emailId) {
+		super();
+		this.id = id;
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this.emailId = emailId;
+	}
+
+	public int getId() {
```

Comment:
Remove getters and setters and use @Getter and @Setter annotations from lombok plugin..

Reply:
okey